mail/chrome/messenger/prefs.properties

```diff
@@ -54,3 +54,6 @@ identity-delete-confirm-button=Usuń
 choosefile = Wybierz plik
 
 forAccount = Zaawansowane ustawienia konta „%S”
+
+removeFromServerTitle=Potwierdzenie trwałego i jednocześnie automatycznego usuwania wiadomości
```

Comment:
I copied it directly from suite. "

"Potwierdzenie trwałego i automatycznego usuwania wiadomości" or "Potwierdzenie trwałego, automatycznego usuwania wiadomości"?